src/lib/types/model.ts

```diff
@@ -386,11 +386,11 @@ export interface IProject {
     defaultStickiness: string;
 }
 
-export interface ICustomRole {
-    id: number;
-    name: string;
+/**
+ * Extends IRole making description mandatory
+ */
```

Comment:
I would still like to drop `ICustomRole` and only have `IRole`. Description is already required in the frontend interface `frontend/src/interfaces/role.ts` and AFAIK there's always a description. 

Could be out of scope for this PR and maybe belong in a separate one though.

Reply:
:+1: but I thought chasing down why the description was optional required a deeper investigation and analyze potential backward incompatibility problems as I see that some project role in our sandbox instance were created without a description